package.json

```diff
@@ -8,8 +8,10 @@
     "node": "4.1.x"
   },
   "scripts": {
-    "start": "node index.js",
-    "test": "mocha test/*_test.js test/**/*_test.js"
+    "start": "node -r babel-core/register -r dotenv/config lib/index.js",
+    "prepublish": "babel lib -d dist",
```

Comment:
In this case, you're probably right that we need a postinstall in order for it to work in heroku. But we don't necessarily need it built locally, so perhaps the better option is to have a `build` script here and we modify the Procfile to use `web: npm run build && npm start` ???